frontend/src/Ux/BuyPhoneCard.tsx

```diff
@@ -0,0 +1,80 @@
+import React, {Component} from "react";
+import {BrowserRouter as Router, Route} from "react-router-dom";
+
+export class BuyPhoneCard extends Component<any, any> {
+    constructor(props: any) {
+        super(props);
+        this.state = {
+            valueName: {},
+            valueSurname: {},
+            valueAmount: {},
+            data: "",
+            isResponse: false,
+            error : false
+        };
+
+        this.getName = this.getName.bind(this);
+        this.getSurname = this.getSurname.bind(this);
+        this.getAmount = this.getAmount.bind(this);
+    }
+
+    private getName(event: any): void {
+        this.setState({valueName: event.target.value, isResponse: false, error : false});
+    }
+
+    private getSurname(event: any): void {
+        this.setState({valueSurname: event.target.value, isResponse: false, error : false});
+    }
+
+    private getAmount(event: any): void {
```

Comment:
Why don't you take advantage of using TypeScript and use correct types instead of "any"?